src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double KOEF_OF_GROW = 1.5;
+    private Object[] array;
```

Comment:
you better call array by name of data that you store inside, in this case it can be items or values